services/director-v2/tests/unit/conftest.py

```diff
@@ -250,9 +265,7 @@ def mocked_storage_service_fcts(fake_s3_settings) -> Iterator[respx.MockRouter]:
         yield respx_mock
 
 
-@pytest.fixture(params=list(FileLinkType))
-def tasks_file_link_type(request) -> FileLinkType:
-    return request.param
+# MOCKS the CATALOG service API responses ----------------------------------------
```

Comment:
hmm this comment is probably out of place

Reply:
no, no ... it groups all fixtures used to mock the catalog service responses